common/dynasties/wc_easter_egg_dynasties.txt

```diff
@@ -0,0 +1,19 @@
+﻿easteregg_dyn1 = {
```

Comment:
Errors:
```
[20:23:56][characterhistory.cpp:454]: 378 is not landed and cannot be set as employer of easteregg7 in 580.1.1 the command won't execute.
[20:23:56][characterhistory.cpp:454]: 378 is not landed and cannot be set as employer of easteregg8 in 582.1.1 the command won't execute.
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Garith' for character 'easteregg2'
[20:23:56][scriptedcharacterlookup.cpp:76]: Duplicate character id in history: centaur176
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Evelyn' for character 'easteregg6'
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Ion' for character 'easteregg9'
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Fenarith' for character 'easteregg4'
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Arren' for character 'easteregg5'
[20:23:56][scriptedcharacterlookup.cpp:76]: Duplicate character id in history: nerubian50
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Derthon' for character 'easteregg7'
[20:23:56][characterhistory.cpp:698]: Missing loc for name 'Melyina' for character 'easteregg8'
```

Reply:
I fixed the loc errors but something is wrong with `employer` not working as it should. **ID 378** _is_ landed and should work fine (it works for Evelyn) but for some reason, it doesn't. Derthon and Melyina are supposed to be in Redridge with their mother, not in their father's court.

Reply:
> I fixed the loc errors but something is wrong with `employer` not working as it should. **ID 378** _is_ landed and should work fine (it works for Evelyn) but for some reason, it doesn't. Derthon and Melyina are supposed to be in Redridge with their mother, not in their father's court.

Fixed in 27f0d3183db856ca726b9127878ae159542db3d9.